hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -267,6 +278,15 @@ export const PendingTables = () => {
                                 </Button>
                             </div>
                         )}
+                        <PopupModal
```

Comment:
Your component should looks the same or almost the same as the Figma design, but right now your design is quite different from it.

Reply:
![image](https://user-images.githubusercontent.com/50423786/202933505-a3d6e56a-2744-43c5-8fe7-8cf93b560707.png)

I think this looks better and makes more sense than the figma. We don't need to provide a list of reasons for why someone should cancel. If they want to, then they should lol

Reply:
Yes it looks nice. But it displays like this on my end.
<img width="1512" alt="CleanShot 2022-11-20 at 22 05 28@2x" src="https://user-images.githubusercontent.com/36619969/202954644-69dfbcaf-4546-4b2d-bbde-44e9411164f2.png">

Reply:
can you git pull?

Reply:
It cant possibly show different text lol

Reply:
<img width="1508" alt="CleanShot 2022-11-20 at 23 02 07@2x" src="https://user-images.githubusercontent.com/36619969/202963010-368a189c-20a2-4915-86a7-184cc60b80b4.png">

git issue you're right! But the size of the modal is still not right on my end.

Reply:
I cannot reproduce your error, I have also accounted for various device sizes already. This was in the original popupModal PR
![image](https://user-images.githubusercontent.com/50423786/202963872-900e0e95-c3f0-47f2-a36a-447b894f85ed.png)
![image](https://user-images.githubusercontent.com/50423786/202963897-e72a4a5b-67ab-4fa7-a275-435688a844f8.png)

Reply:
![image](https://user-images.githubusercontent.com/50423786/202964189-db26416e-2d5b-485c-b24e-190e2355a18e.png)

Reply:
Problem Resolved. Due to Leo's browser cache